app/components/Ui/Input.vue

```diff
@@ -1,9 +1,15 @@
 <template>
-  <!-- eslint-disable-next-line vue/html-self-closing -->
-  <input v-bind="props" v-model="localModel" :class="styles({ class: props.class })" />
+  <input
+    v-bind="props"
+    :value="modelValue"
+    @input="handleInput"
+    :class="styles({ class: props.class })"
+  />
 </template>
 
 <script lang="ts" setup>
+  import { computed } from "vue";
```

Comment:
Removing this as it is not needed in nuxt

Reply:
Thanks again @arshx86 

Appreciate it. The cli will be updated with the changes soon.